### DISCUSSION TITLE
内置编辑器代码高亮问题

### DISCUSSION THREAD

**Discussion by genment** [1Panel-dev/1Panel#5560] [Q&A]
### 1Panel 版本

v1.10.10-lts

### 问题描述

内置编辑器代码高亮错误地将 `/*` 识别为注释的开始标记，导致 `/*` 后面的所有内容都变灰了。

### 重现步骤

如图所示。

不止这里，只要文件类型不支持 `/* ... */ ` 注释的配置文件，都会出现这个问题，比如 docker、openresty/nginx 的配置文件

### 期待的正确结果

在配置文件中，`/*` 不应该被识别为注释标记

### 附加信息

![image](https://github.com/1Panel-dev/1Panel/assets/5207957/e36f0aa7-918f-4f54-8c0a-d1662b22df23)

**Comment by wanghe-fit2cloud** [comment]
目前配置文件编辑器使用的是 [vue-codemirror](https://github.com/orgs/codemirror/repositories?q=lang-&type=all)，支持高亮的语言库目前不包括 nginx 等。
